docs/REST API/rest-api.md

```diff
@@ -8122,23 +8122,39 @@ Content-Type: application/octet-stream
 `Status: 200`
 
 
-### Download software
+### Get or download software
 
 _Available in Fleet Premium._
 
-Download a software package.
+Get info or download a software package.
 
-`GET /api/v1/fleet/software/titles/:software_title_id/package/?alt=media`
+`GET /api/v1/fleet/software/titles/:software_title_id/package`
 
 #### Parameters
 
 | Name            | Type    | In   | Description                                      |
 | ----            | ------- | ---- | --------------------------------------------     |
 | software_title_id              | integer | path | **Required**. The ID of the software title to download software package.|
 | team_id | integer | form | **Required**. The team ID. Downloads a software package added to the specified team. |
-| alt             | integer | query | **Required**. If specified and set to "media", downloads the specified software package. |
+| alt             | integer | query | If specified and set to `media`, downloads the specified software package. |
+| token | string | query | Download with authentication token. `alt` must be specified as `media` to download with the token. |
 
-#### Example
+#### Example (get package info)
+
+`GET /api/v1/fleet/software/titles/123/package`
+
+##### Default response
+
+`Status: 200`
+
+```json
+{
+  "name": "ZoomFullInstaller.pkg"
+  "token": "0331f58a-5bf8-4eb6-959d-f1dbd7fa8d64"
```

Comment:
token shouldn't live in info. It's a short lived one time use secret so I think it's more appropriate to only get a token when requesting with `alt=token` IMO

Reply:
We don't want every get info to store a token that may not be needed.